tests/src/test/resources/com/powsybl/openrao/tests/features/epic12_export_improvement/US12_15.feature

```diff
@@ -82,14 +82,9 @@ Feature: US 12.15: export different reason per perimeter in SWE CNE
     Given crac creation parameters file is "epic12/CimCracCreationParameters_MonitorLeftSide.json"
     Given configuration file is "epic12/raoParametersSweIDCC_minObjectiveDisabled2P.json"
     When I launch search_tree_rao at "2021-04-02 05:00"
-    Then the calculation partially fails
```

Comment:
Not sure why exactly all those tests are impacted in the same way. Maybe @phiedw it would be good to explicitly understand why.